<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Organize Files</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>300</height>
   </size>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">

    <!-- Path selection -->
    <item>
     <layout class="QHBoxLayout" name="pathLayout">
      <item>
       <widget class="QLineEdit" name="linePath">
        <property name="placeholderText">
         <string>ุงุฎุชุฑ ูุณุงุฑ ุงููุฌูุฏ...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBrowse">
        <property name="text">
         <string>๐ ุชุตูุญ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <!-- Start button -->
    <item>
     <widget class="QPushButton" name="btnStart">
      <property name="minimumHeight">
       <number>36</number>
      </property>
      <property name="text">
       <string>โถ ุงุจุฏุฃ ุงูุชูุธูู</string>
      </property>
     </widget>
    </item>

    <!-- Pause + Theme -->
    <item>
     <layout class="QHBoxLayout" name="controlLayout">
      <item>
       <widget class="QPushButton" name="btnPause">
        <property name="text">
         <string>โธ ุฅููุงู ูุคูุช</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTheme">
        <property name="text">
         <string>๐ Dark / Light</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <!-- Progress bar -->
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>

    <!-- Status -->
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>ุงูุญุงูุฉ: ุฌุงูุฒ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>

   </layout>
  </widget>

  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>

 <resources/>
 <connections/>
</ui>
